project.py

```diff
@@ -134,9 +134,9 @@ def area_code_lookup(phone_nums:str) -> dict:
     return output_dict
 
 
-def get_state_abbrev() -> dict:
+def get_state_abbrev(reverse=False) -> dict:
     """Returns a dict of state abbreviations and corresponding states"""
-    return {
+    states = {

```

Comment:
great job converting the results to a variable called states instead of just returning